dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java

```diff
@@ -153,10 +153,13 @@ public GHContent tryRetrievingContent(GHRepository repo, String path, String bra
      * OkUrlFactory, which is required to specify the cache. In other words, we cannot flush the cache.
      *
      * Instead, we wait for 60 seconds if the list retrieved is not the list we want.
+     *
+     *
+     * TODO: Did updating the GitHub api fix this?
```

Comment:
Is this actionable? Feels like this TODO will be there forever :)

Reply:
Fair 😄